### ISSUE TITLE
An app that can function as a video player for Stremio on iOS (not requesting a fully featured app)

### ISSUE THREAD

**Issue by FlashLightning67** [Stremio/stremio-features#660]
**Is your feature request related to a problem? Please describe.**
Due to the limitations of using Stremio web without Stremio service, using Stremio on iOS lacks features such as being able to track progress in content to pick up where you left off. 

**Describe the solution you'd like**
An app (or update to Stremio organizer) that can function as an external player with Stremio integrated. This can be completely separate from the rest of Stremio, which would all stay in the web app, so it would hopefully would be allowed by Apple as it would essentially just be a video player like VLC or Outplayer. However this player could keep track of things for Stremio that usually can’t be tracked due to content playing in a third party player, and that could all be used as it would on other OS’s to track progress in a movie/episode and what not. This app would be another option in the external players drop down, and we would still use the web app for all the functions that we normally do, this app would just replace VLC and Outplayer. 

In other words this would be an app that could provide more features that the web app can alone by moving some features that can’t be provided by the web app over to a standalone app that only does things that Apple likes.

**Comment by TRtomasz** [comment]
Stremio organizer will not receive any new features, sorry.
